### ISSUE TITLE
[koffi]  ctrl+c not working when use koffi + npm run dev + electron + vite + typescript

### ISSUE THREAD

**Issue by joyexpr** [Koromix/rygel#60]
normally, when I  run (npm run dev)  a  electron + vite + typescript appliction,  if I pressed ctrl + c on terminal will kill the app.

but after I  import  koffi, when I pressed ctrl + c, then app will not be killed anymore, I need to kill the app by using process manager of os.

I only add some lines below:

```
import koffi from 'koffi'
function loadDll(path) {
  koffi.load(path)
}
```

I found it happened on macos and linux, but windows 10 is normal

**Comment by joyexpr** [comment]
minimal example below:  
yarn install  or npm install
npm run dev
ctrl + c  

https://obs-lego-devops.obs.cn-east-2.myhuaweicloud.com:443/test/koffi-bug2.zip?AccessKeyId=P5RGESDE95NJKTWNB7MV&Expires=1689755598&Signature=xYUrvU37q9Vvak0kxetnH9/Tkp8%3D

**Comment by Koromix** [comment]
Should be fixed in Koffi 2.5.0 which is in beta.
You can try beta1 with `npm install koffi@beta`.

**Comment by joyexpr** [comment]
I've verified it.  great ! thank you
